app/views/completed_applications/_return_notification.html.erb

```diff
@@ -18,6 +18,6 @@
 
     <%= button_to recreate_completed_crime_application_path, method: :put,
                   class: 'govuk-button app-button--blue govuk-!-margin-bottom-2 app-no-print',
-                  data: { module: 'govuk-button' } do; t('.update_application'); end %>
+                  data: { module: 'govuk-button' } do; t('.add_justification'); end %>
```

Comment:
So this should only be 'add_justification' if the return reason is add justification. Otherwise, for all other return reasons, update_application is correct.